tiledb/sm/rtree/rtree.cc

```diff
@@ -385,7 +380,8 @@ RTree::Level RTree::build_level(const Level& level) {
 
   uint64_t mbrs_visited = 0, offset_level = 0, offset_new_level = 0;
   for (uint64_t i = 0; i < new_level.mbr_num_; ++i) {
-    auto mbr_num = MIN(fanout_, level.mbr_num_ - mbrs_visited);
+    auto mbr_num =
+        std::min<long unsigned>(fanout_, level.mbr_num_ - mbrs_visited);
```

Comment:
We _try_ to use the cstdint typedefs for primitives, please use `uint64_t` here instead of `long unsigned`.